src/index.html

```diff
@@ -6,9 +6,88 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="headTable">
+      <a class="logo" href="#home">
+        <img
+        src="./images/logo.png"
+        class="logo"
+        alt="logo"
+        >
+      </a>
+      <nav class="container">
+        <ul class="nav__list ">
+          <li class="nav__item">
+            <a
+              class="is-active nav__link"
+              href="#"
+            >
+            Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              >
+              Samsung
+            </a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a
              class="is-active nav__link"
              href="#"
            >
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a
                href="#"
                class="nav__link"
              >
                Samsung
            </a>
          </li>
          
```
check your code style
add blank lines between children, and check indents